pandas/tests/frame/test_reductions.py

```diff
@@ -1025,6 +1025,28 @@ def test_idxmax_mixed_dtype(self):
         expected = Series([0, 2, 1, 2], index=[1, 2, 3, 4])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "func_name, expected_value",
+        [("idxmax", [0, 4]), ("idxmin", [0, 5])],
+    )
+    def test_idxmax_idxmin_convert_dtypes(self, func_name, expected_value):
+        df = DataFrame(
```

Comment:
Add comment for the issue as the first line of the test, e.g. # GH 40346

Reply:
thanks @rhshadrach , I'll put in these changes.